### PR TITLE
WIP Only add one button if called from outside

### PR DESCRIPTION
**Pull request by Gomez**
I need to call Faceted.Events.INITIALIZE from the outside of eea. This fixes creation of more than one "More" buttons

### CODE REVIEW

File: eea/facetednavigation/browser/javascript/expand.js [hunk 1]
```diff
@@ -73,26 +73,28 @@ jQuery.fn.collapsible = function(settings){
...
-        .hide();
-      self.append(self.button);
-
-      link.click(function(){
-        if(self.colapsed){
-          self.trigger(options.events.expand);
-        }else{
-          self.trigger(options.events.colapse);
+      // Only add button if none exits
+      if (jQuery('.faceted-checkbox-more').length < 1){
```
Comment (avoinea) [inline eea/facetednavigation/browser/javascript/expand.js:77]: @Gomez :thinking: Hmm, what do you do if you have 2 or more checkbox widgets on the same page?

Comment (Gomez) [reply eea/facetednavigation/browser/javascript/expand.js:77]: Yeah, its not working and i found another bug :(

Is there a way to reset the eea state with a reset button from outside? Without reload.

Comment (avoinea) [reply eea/facetednavigation/browser/javascript/expand.js:77]: @Gomez I think you can write directly in Faceted.Query :thinking: and then trigger some events:

https://github.com/eea/eea.facetednavigation/blob/bb85f41f45ea26561c0e306a13975f5b02a0e6d7/eea/facetednavigation/browser/javascript/view.js#L184

See if there is one that can help:

https://github.com/eea/eea.facetednavigation/blob/bb85f41f45ea26561c0e306a13975f5b02a0e6d7/eea/facetednavigation/browser/javascript/view.js#L5-L20

Comment (avoinea) [reply eea/facetednavigation/browser/javascript/expand.js:77]: Maybe this one:

```javascript
jQuery(Faceted.Events).trigger(Faceted.Events.RESET); 
```

Comment (Gomez) [reply eea/facetednavigation/browser/javascript/expand.js:77]: Thanks! "QUERY_INITIALIZED" was the missing piece. 

This works:

```
$('.button.reset').on('click', function (e) {
    jQuery(Faceted.Events).trigger(Faceted.Events.RESET);
    Faceted.Query.b_start = 0;
    Faceted.b_start_changed = true;
    Faceted.URLHandler.set();

   jQuery(Faceted.Events).trigger(Faceted.Events.QUERY_INITIALIZED);
}
```

**Pull request closed without merging**